gosam.spec

```diff
@@ -12,6 +13,7 @@ Requires: python3 py3-cython
 %prep
 %setup -q -n %{n}-%{realversion}
 sed -i -e 's|^VERSION =.*|VERSION = "%{realversion}"|;s|^GIT_REVISION =.*|GIT_REVISION = "%{tag}"|' setup.py
+%patch0 -p1
```

Comment:
@iarspider , why was this change needed?

Reply:
@smuzaffar `setuptools` complains that `2.1.0-2.1.0` is a malformed verison

Reply:
ah ok @iarspider . I guess there is no point having an extra patch for `setup.py` and also using `sed` to update it.   I would suggest to just update the `sed -i -e ` command above and fix the  `TAR_VERSION` too e.g.
```
sed -re 's#^((VERSION|GIT_REVISION|TAR_VERSION) *= *).*#\1"%{realversion}"#' setup.py
```

also add the following before the `sed` command
```
grep -q '^VERSION *=' setup.py && grep -q '^GIT_REVISION *=' setup.py && grep -q '^TAR_VERSION *=' setup.py
```